lib/src/rules.dart

```diff
@@ -103,6 +103,7 @@ import 'rules/no_leading_underscores_for_library_prefixes.dart';
 import 'rules/no_leading_underscores_for_local_identifiers.dart';
 import 'rules/no_logic_in_create_state.dart';
 import 'rules/no_runtimeType_toString.dart';
+import 'rules/no_self_assign.dart';
```

Comment:
We haven't been totally consistent but I think we've standardized more on pluralized names.  In which csae, how about:

    no_self_assignments

Or, maybe:

    self_assignments

?

Reply:
Done `no_self_assignments`

Reply:
Thank you!